examples/adminapi.py

```diff
@@ -419,17 +420,92 @@ def example_list(a, args):
         print(" {} consumer groups".format(len(groups)))
         for g in groups:
             if g.error is not None:
-                errstr = ": {}".format(t.error)
+                errstr = ": {}".format(g.error)
             else:
                 errstr = ""
 
             print(" \"{}\" with {} member(s), protocol: {}, protocol_type: {}{}".format(
-                  g, len(g.members), g.protocol, g.protocol_type, errstr))
+                g, len(g.members), g.protocol, g.protocol_type, errstr))
 
             for m in g.members:
                 print("id {} client_id: {} client_host: {}".format(m.id, m.client_id, m.client_host))
 
 
+def example_delete_consumer_groups(a, args):
+    groups = a.delete_consumer_groups(args, timeout=10)
+    for group_id, future in groups.items():
+        try:
+            response = future.result()
+            print("Deleted group with id '" + group_id + "' succesfully")
+
+        except KafkaException as e:
+            print("Failed to delete group '{}': {}".format(group_id, e))
+        except Exception:
+            raise
+
+
+def example_list_consumer_group_offsets(a, args):
+    """ List consumer group offsets
+    """
```

Comment:
Yep, you can nick the clang-format style script from librdkafka: https://github.com/edenhill/librdkafka/blob/master/packaging/tools/style-format.sh

(can be simplified since there is no mix of C and C++ in this repo)

Also make sure to add style-checking to CI so that PRs are rejected if style is not followed.
But as Matt said, in a later PR.

Reply:
Makes sense. Created ticket for the same. [CLIENTS-2999](https://confluentinc.atlassian.net/browse/CLIENTS-2999)